tests/warp/warp_test.go

```diff
@@ -265,20 +266,6 @@ func (w *warpTest) initClients() {
 	}
 }
 
-func (w *warpTest) getBlockHashAndNumberFromTxReceipt(ctx context.Context, client *ethclient.Client, tx *types.Transaction) (common.Hash, uint64) {
-	// This uses the coreth client to fetch a block from Coreth (when testing the C-Chain), so we use this
-	// workaround to get the correct block hash. Note the client recalculates the block hash locally, which results
-	// in a different block hash due to small differences in the block format.
-	require := require.New(ginkgo.GinkgoT())
-	for {
-		require.NoError(ctx.Err())
```

Comment:
(fyi) The correct usage of a context here is with `ctx.Done()`, which returns a channel that will be closed when either `cancel()` is called or the deadline/timeout is reached. In this specific case it's unnecessary because `client.TransactionReceipt()` is responsible for that (perhaps a few function calls down though; my guess is at the HTTP layer).